stm/asyncTaskQueue.ml

```diff
@@ -123,7 +123,7 @@ module Make(T : Task) = struct
                  "-worker-id"; name;
                  "-async-proofs-worker-priority";
                    Flags.string_of_priority !Flags.async_proofs_worker_priority]
-        | ("-ideslave"|"-emacs"|"-emacs-U"|"-batch")::tl -> set_slave_opt tl
```

Comment:
The option is already marked as obsolete below; however PG may need a minor (one line) patch.